### PR TITLE
Verification for plots targeted in Immunogenicity Report

### PR DESCRIPTION
**Pull request by dilu963**
Verification is done for plots targeted for independent double programming in Immunogenicity Report.

### CODE REVIEW

**Comment by benkeser** [comment]
Please remove all `.rds` `.png` `.PNG` `.csv`  files from the PR and resubmit. These can be added to a `.gitignore` file or deleted.

The `renv` files also look to be modified questionably modified in places. @nhejazi want to double check me on this?

**Comment by benkeser** [comment]
> Please remove all `.rds` `.png` `.PNG` `.csv` files from the PR and resubmit. These can be added to a `.gitignore` file or deleted.

You'll need to [explicitly remove these files after adding to `gitignore`](https://superuser.com/questions/1397199/how-to-ignore-a-tracked-file-in-git-without-deleting-it)

**Comment by dilu963** [comment]
Sorry, I am still trying to get familiar with GitHub. Do you mean remove these files and then recommit?

**Comment by benkeser** [comment]
Can you also remove `.Rproj` and `renv.lockfile`?

**Comment by benkeser** [comment]
@dilu963 looks like some merge conflicts to fix in `.gitignore`

@thebioengineer can you check that this is all good?

**Comment by thebioengineer** [comment]
Sure thing. I'll take a look and get back to you!

File: immuno_graphical/verification/verification_plan.Rmd [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ---
-title: Verification Plan for Immunogenicity Plots
-date: "`r format(Sys.time(), '%B %d, %Y')`"
-output: pdf_document
+title: "Verification Plan for Immunogenicity Plots"
+output:
+  html_document: default
```
Comment (thebioengineer) [inline immuno_graphical/verification/verification_plan.Rmd:4]: hey @dilu963 - could you remove html_document as an output? and the html document that you produced?

File: cor_graphical/verification/verification_report.Rmd [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+title: 'Verification Report: correlates_graphical'
+author: "Di Lu"
+date: "2021/03/27"
+output:
+  html_document: default
```
Comment (thebioengineer) [inline cor_graphical/verification/verification_report.Rmd:6]: Please remove the html_document output and keep only the pdf_document output. Remove html files that were generated.

File: cor_graphical/verification/verification_report.Rmd [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+This is the verification report for the `cor_graphical` folder of the `correlates_reporting` project for CoVPN. 
+
+In the verification process, the output of `cor_graphical_verification_code.R` is compared against the plots listed in the the folder `immuno_graphical/verification/verification_input`. The values presented in the original plot against the ones generated by the verification code. Any statistical calculations will either be verified by computing them to a verified statistical table , or they will be generated independently.
+
+The files `cor_graphical/verification/verification_input/practice_data.csv`, `cor_graphical/verification/verification_input/longer_cor_data_plot1.rds` and 
+`cor_graphical/verification/verification_input/plot.25sample1.rds` was provided by the original programmer to the tester for verification purposes.
+
+####lineplots of Binding Antibody to Spike: baseline negative vaccine arm (2 timepoints)
+Outputs of Line plots, Violin plots and box plots and were visually compared to the outputs generated by primary programmer, one issue that was identified was the titer values were turncated by wrong ULOQ, so the verification is pending.
```
Comment (thebioengineer) [inline cor_graphical/verification/verification_report.Rmd:20]: Please add a programmatic comparison of the input data & calculated statistics you generated against the data 
 & stats provided by kendrick. (save the input data for each plot you generated against the input data kendrick generated for each plot and compare via `identical()` or `dplyr::all_equal()`.

**Review by thebioengineer — CHANGES_REQUESTED** [review]
Hey @dilu963 - this is looking great. Lets have a call tomorrow morning to smooth out the merge conflicts and add progammatic comparisons into the verification reports!

**Pull request closed without merging**